content/ember-released-6-8.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Ember 6.8 Released
+authors:
+  - jared-galanis
+date: 2025-10-15T00:00:00.000Z
+tags:
+  - releases
+  - '2025'
+  - version-6-x
+---
+
+Today the Ember project is releasing version 6.8 of Ember.js and Ember CLI. This release of Ember.js is an LTS (Long Term Support) candidate. LTS candidates prioritize stability over the addition of new features, and have an extended support schedule.
+
+A release of Ember is comprised of many projects: ember-source, ember-cli, and all the learning team projects. We follow a [6-week release train](http://emberjs.com/releases/) that includes alpha and beta cycles to ensure changes are well-tested. We encourage our community (especially addon authors) to help test the beta builds and report any bugs before they are published as a final release. The `ember-try` addon is a great way to continuously test your projects against the latest Ember releases. While we only consider the release to be complete upon publication of the blog post, the 6-week cycle is anchored by the release of the ember-source package at the beginning of the process. Releases x.4, x.8 and x.12 will become an LTS candidate, so checkout [Ember LTS Releases](https://blog.emberjs.com/announcing-embers-first-lts/) if you want to know more.
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 6.8
+
+Ember.js 6.8 is an incremental, backwards compatible release of Ember with bug fixes, performance improvements, and minor deprecations.
+
+#### Bug fixes
+
+Ember.js 6.8 introduced <insert number here> bug fixes.
+
+#### Features
+
+Ember.js 6.8 introduces <insert number here> new features.
```

Comment:
- renderComponent
- `@ember/reactive/collections`

Will PR some more verbiage for these shortly